channels/channel.be/vier/chn_vier.py

```diff
@@ -614,6 +622,10 @@ def __extract_stream_collection(self, item, video_info):
                 if "/geo" in stream_url:
                     item.isGeoLocked = True
 
+            if not streams:
```

Comment:
Even met line 601 meegenomen: is het niet gewoon overal dan de "long-form"-url zijn?

Reply:
Ik wilde het oude er eerst in houden voor het geval dat (ook gezien https://github.com/retrospect-addon/plugin.video.retrospect/pull/1820#discussion_r1700872727), maar het lijkt nergens gebruikt te worden.